src/store/reducers/storage.ts

```diff
@@ -127,6 +129,9 @@ const storageSlice = createSlice({
     setAutoUpdateEnabledBeforeDowngrade: (state, action: PayloadAction<boolean>) => {
       state.wasAutoUpdateEnabledBeforeDowngrade = action.payload;
     },
+    setSecondaryNetworkWalletCreated: (state, action: PayloadAction<boolean>) => {
```

Comment:
Maybe have an object for networks instead? So if we have more than just one testnet